src/image-button/image-button.style.tsx

```diff
@@ -0,0 +1,60 @@
+import { MediaQuery } from "../media";
+import styled, { css } from "styled-components";
+import { Color } from "../color/color";
+import { DesignToken } from "src/design-token/design-token";
+
+interface StyleProps {
+    $outline: "none" | "browser";
+    $disabled?: boolean;
+}
+
+export const Main = styled.button<StyleProps>`
+    display: flex;
+    flex-direction: column;
+    gap: 0.5rem;
+    justify-content: center;
+    align-items: center;
+    padding: 1rem 1.5rem;
+    background-color: transparent;
+    border: none;
+    border-radius: 0.5rem;
+    cursor: pointer;
+    width: 13rem;
+    height: 8.625rem;
+    transition: all 200ms ease;
+    img {
+        width: 3.5rem;
+    }
+    ${MediaQuery.MaxWidth.tablet} {
+        padding: 0rem 1.5rem;
+    }
+
+    ${MediaQuery.MaxWidth.mobileM} {
+        padding: 0rem 1rem;
+    }
+    ${(props) => {
+        if (props.$disabled) {
```

Comment:
there's no need for a separate $disabled prop. buttons should have the `:disabled` pseudo class

Reply:
Done